packages/story-editor/src/components/panels/document/publish/publish.js

```diff
@@ -189,11 +189,20 @@ function PublishPanel() {
           isHighlighted={
             highlightPoster?.showEffect || highlightLogo?.showEffect
           }
+          onAnimationEnd={() => resetHighlight()}
         >
           <MediaInputWrapper>
             <MediaWrapper>
               <StyledMedia
-                ref={posterButtonRef}
+                ref={(node) => {
+                  if (
+                    node &&
+                    highlightPoster?.focus &&
+                    highlightPoster?.showEffect
+                  ) {
+                    node.focus();
+                  }
+                }}
```

Comment:
These two functions (this and the ref setter below) are basically identical - couldn't they be generalized?

Reply:
Will de-dupe in the follow up!